documentation/man/features/init.rst

```diff
@@ -6,8 +6,8 @@ kw-init
 
 SYNOPSIS
 ========
-| *kw* *init* [(-v | \--verbose)] [\--template[=name]]
-| *kw* *init* [(-v | \--verbose)] [\--arch <name>][\--remote <user>@<ip>:<port>][\--target <target>]
```

Comment:
You don't need this extra commit for that; go back to the previous commit and just remove `-v` from there.